examples/lighting-app/lighting-common/lighting-app.matter

```diff
@@ -2354,8 +2355,16 @@ endpoint 0 {
   }
 
   server cluster Switch {
+    emits event SwitchLatched;
+    emits event InitialPress;
```

Comment:
A Switch cluster can't have both LS and MS features at once.

Reply:
Acked, removed MS feature